### PR TITLE
Add confirmation to admin UI for email delivery

### PR DESCRIPTION
**Pull request by joshuatobin**
Adds some visual confirmation to the UI if an email was actually sent out, and removes the 'Sent Retro Email' button if so.

screen shot 2016-11-15 at 1 03 21 pm

CC: @reidmix

### CODE REVIEW

File: app/admin/incident.rb [hunk 1]
```diff
@@ -126,7 +134,7 @@ class TrelloAuthRequired < StandardError; end
           { To:      :text,
             CC:      :text,
             Subject: :text
-          }.to_json
+          }.to_json unless incident.retro_email_sent?
```
Comment (ys) [inline app/admin/incident.rb]: I find it hard to see what is the scope of the unless here... with the two levels of hashes and different syntax.

File: db/schema.rb [hunk 2]
```diff
@@ -53,6 +53,7 @@
     t.string   "google_doc_url"
     t.datetime "retro_at"
     t.integer  "primary_team_id"
+    t.boolean  "retro_email_sent"
```
Comment (ys) [inline db/schema.rb]: Please add a default to false here. You have a three legs boolean.

File: db/schema.rb [hunk 3]
```diff
@@ -42,8 +42,8 @@
     t.integer  "incident_id"
     t.boolean  "review"
     t.date     "followup_on"
-    t.datetime "created_at",      null: false
-    t.datetime "updated_at",      null: false
+    t.datetime "created_at",       null: false
```
Comment (joshuatobin) [inline db/schema.rb]: @ys Not sure. This is automated AFAICT

File: db/schema.rb (hunk 2 above)
Comment (joshuatobin) [inline db/schema.rb]: Fixed in ae30557 @ys

File: app/admin/incident.rb (hunk 1 above)
Comment (joshuatobin) [inline app/admin/incident.rb]: Not sure it reads as well but fixed in 0c6f277

**Comment by joshuatobin** [comment]
@ys Addressed all comments.

**Pull request merged**